jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/MockAwsConnectionManager.kt

```diff
@@ -75,7 +75,7 @@ class MockAwsConnectionManager(project: Project) : AwsConnectionManager(project)
     }
 }
 
-fun <T> runUnderRealCredentials(project: Project, block: () -> T): T {
+inline fun <T> runUnderRealCredentials(project: Project, crossinline block: () -> T): T {
```

Comment:
Didn't know that was a keyword, interesting

Reply:
We actually dont need it, we dont pass the block off to another function context